kano_updater/ui/views/finish.py

```diff
@@ -62,3 +63,6 @@ def _finish(self, window=None, event=None):
             os.system('systemctl poweroff')
         else:
             os.system('systemctl reboot')
+
+        time.sleep(10)
+        os._exit(0)
```

Comment:
Oh yes. I was concerned that Gtk would not like a normal `sys.exit()`, I'll push a change and try another round. It's the only way to test it because the updater program updates its own package and relaunches itself in the process.

[os._exit()](https://docs.python.org/2/library/os.html#os._exit)